### PR TITLE
Make category/type options correct in editors for previous years

### LINKED ISSUE #137: The type and category in the proposal editor are not filled in when you open proposal editor for previous year
Because the editor contains fields for the current year rather than the year the workshop was from.

These editors should be locked down when looking at previous years anyway...

### PR DESCRIPTION
**Pull request by krzys-h**
This part can be merged now though since it's a bugfix. See #143.

Closes #137

### CODE REVIEW

**Comment by krzys-h** [comment]
Fixed merge conflict

File: wwwapp/forms.py [hunk 1]
```diff
@@ -177,6 +177,11 @@ def __init__(self, *args, **kwargs):
         self.helper.form_tag = False
         self.helper.disable_csrf = True
         self.helper.include_media = False
+
+        year = self.instance.type.year if hasattr(self.instance, 'type') else settings.CURRENT_YEAR
+        self.fields['category'].queryset = WorkshopCategory.objects.filter(year=year)
```
Comment (cytadela8) [inline wwwapp/forms.py]: Remove queryset from lines 167 & 171

Comment (krzys-h) [reply wwwapp/forms.py]: You can't - it's a required constructor argument :/

Comment (cytadela8) [reply wwwapp/forms.py]: :'(

**Pull request merged**